apps/dashboard/src/components/settings/ApiKeys/validations.ts

```diff
@@ -149,4 +149,4 @@ export type ApiKeyPayConfigValidationSchema = z.infer<
 >;
 
 // FIXME: Remove
-export const HIDDEN_SERVICES = ["relayer", "chainsaw", "insight"];
+export const HIDDEN_SERVICES = ["relayer", "chainsaw"];
```

Comment:
any dependencies for this? do we need to show special UI for it or something?

Reply:
Currently no. The api key will be used to query the insight API.